app/config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   port: process.env.PORT || 3000,
 
   // The base uri the app is running on to pass to identity service for redirection
-  appBaseUri: process.env.APP_BASE_URI || 'http://localhost:3000',
+  appBaseUri: process.env.APP_BASE_URI || 'http://docker.for.win.localhost:3000',
```

Comment:
will this work for mac?

Reply:
It won't - it will need to be http://doker.for.mac.localhost:3000

Given that most (as of today ALL) of our front-end devs work on a mac, I'll make mac the default and anyone working on windows box will need to change that setting :]

Reply:
Ben doesn't

Reply:
But I agree with that idea 😄

Reply:
🤦‍♂ For some idea I thought he had a mac as well. Well... 2 > 1 so I guess it makes sense to have to set to mac. We just need to make sure we communicate this to Ben